src/components/home/Home.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import RelatedCards from "./sections/RelatedCards";
+import "./index.css";
```

Comment:
i don't think you want index.css imported in all these files, they should all have their own css